### ISSUE TITLE
How to retrieve app store url with delegate?

### ISSUE THREAD

**Issue by ndduong97** [ArtSabintsev/Siren#156]
I am trying to use the delegate method sirenDidDetectNewVersionWithoutAlert(message: String) to call my own custom alert view.

I can get the localized text but don't know how to get the url that comes with it. Is there a way I can retrieve the url to update the app?

**Comment by ArtSabintsev** [comment]
Seems that I need to expose this method. It's currently marked as private in a private extension.

```swift
    func launchAppStore() {
        guard let appID = appID,
            let iTunesURL = URL(string: "https://itunes.apple.com/app/id\(appID)") else {
            return
        }

        DispatchQueue.main.async {
            UIApplication.shared.openURL(iTunesURL)
        }
    }
```

**Comment by ndduong97** [comment]
Yeah I just found that method too. Do you think it's possible that you push this out with the next version too? That would be so helpful :)

**Comment by ArtSabintsev** [comment]
Yup!

I'll do it in the #157 PR.

**Comment by ArtSabintsev** [comment]
ok, done: https://github.com/ArtSabintsev/Siren/pull/157/commits/c5daa9d1ca6f888d6e2f143eb35bb7ba5da2334c

Waiting for CI to pass. Afterwards, I'll merge it to master and release a new build. 10 mins max.

**Comment by ArtSabintsev** [comment]
Fixed and available via all package managers as v2.0.7: https://github.com/ArtSabintsev/Siren/releases/tag/2.0.7
